src/presenter/movie-wrapper.js

```diff
@@ -0,0 +1,119 @@
+import EmptyFilmMessageView from '../view/message-empty-film-list';
+import ShowMoreButtonView from '../view/show-more-button';
+import MovieWrapperView from '../view/movie-wrapper';
+import MovieCardPresenter from './movie';
+
+import {
+  renderElement, RenderPosition, remove,
+} from '../utils/render';
+
+import { updateItem } from '../utils/common';
+
+const TASK_COUNT_PER_STEP = 5;
+
+export default class MovieList {
+  constructor(filmContainer, classNameSection, title, containerListAttribute) {
+    this._filmContainer = filmContainer;
+    this._emptyFilmMessage = new EmptyFilmMessageView();
+    this._showMoreButton = new ShowMoreButtonView();
+
+    this._handleLoadMoreButtonClick = this._handleLoadMoreButtonClick.bind(this);
+    this._handleFilmUpdate = this._handleFilmUpdate.bind(this);
+    this._handleModeChange = this._handleModeChange.bind(this);
+
+    this._movieWrapper = new MovieWrapperView(
+      classNameSection,
+      title,
+      containerListAttribute,
+    );
+
+    this._filmsPresenter = {};
+    this._movieWrapperList = null;
+    this._renderedTaskCount = TASK_COUNT_PER_STEP;
+  }
+
+  init(filmCards) {
+    this._renderFilmsContainer();
+
+    this._renderFilmsList(filmCards);
+  }
+
+  sortFilms(films) {
+    this._clearFilmsList();
+    this._renderFilmsList(films);
+  }
+
+  _handleFilmUpdate(updateFilm) {
```

Comment:
If `updateFilm` is not a function then should rename it

```suggestion
  _handleFilmUpdate(updatedFilm) {
```

Reply:
fixed